bedrock/mozorg/templates/mozorg/plugincheck.html

```diff
@@ -55,6 +55,11 @@
   {# L10n: This is the string used to indicate that the plugin is unknown #}
   data-unknown="{{ _('unknown') }}"
 
+  {% if l10n_has_tag('plugin_outdated') -%}
+    {# L10n: This is the string used to indicate that the plugin is outdated #}
+    data-outdated="{{ _('outdated') }}"
+  {% endif %}
```

Comment:
This can definitely be shown frequently, so I would go with the confusing outdate = "vulnerable" as unknown means something completely different in this context. Thanks @flodolo